src/Pin.js

```diff
@@ -1,110 +1,103 @@
-import React, { PureComponent } from "react";
+import React from "react";
 
-export default class Pin extends PureComponent {
-  render() {
-    const { onClick, color, selected } = this.props;
+const ICON_WIDTH = 30;
+const ICON_SELECTED_WIDTH = 47;
+const ICON_HEIGHT = 36;
+const ICON_SELECTED_HEIGHT = 53;
 
-    return (
-      <svg
-        width={selected ? 47 : 30}
-        height={selected ? 53 : 36}
-        viewBox={selected ? "0 0 47 53" : "0 0 30 36"}
-        fill="none"
-        style={{
-          cursor: "pointer",
```

Comment:
Looks like this `cursor: "pointer"` style got dropped along the way - is that the reason I saw it's missing in the preview build?

Reply:
I feel like the library unintentionally is paving over cursor styles passed in:
https://github.com/uber/react-map-gl/blob/5.1-release/src/components/marker.js#L92

And they're just relying on setting the style directly on the child passed in. For reference, this is the example I was using for guidance when building this: https://github.com/uber/react-map-gl/blob/5.1-release/examples/controls/src/city-pin.js

Reply:
I see! I ended up just adding a `svg` selector to the styled component css and it restored the correct cursor: pointer style. Thanks!